### PR TITLE
[MAT-379] patch javadoc bug, this is a PR demo, deliberate mistake\!

### PR DESCRIPTION
**Pull request by stuartarchibald**
This PR is a minor bugfix for javadoc not building with `<li>` tags.
This is a demo PR!

Coverage: No change as textual changes only.

### CODE REVIEW

File: src/java/src/main/java/com/opengamma/maths/datacontainers/matrix/OGComplexDenseMatrix.java [hunk 1]
```diff
@@ -58,8 +58,8 @@ public OGComplexDenseMatrix(double[][] realPart, double[][] imaginaryPart) {
   /**
    * Takes a column major double[] and turns it into an OGComplexArray
    * The length of dataIn must be either:
-   * <li>rows*columns in which case it is assumed the double[] provided is the real part of the data (imaginary part assumed zero)
-   * <li>2*rows*columns in which case it is assumed the double[] provided is formed of interleaved real and imaginary values
+   * a) rows*columns in which case it is assumed the double[] provided is the real part of the data (imaginary part assumed zero)
+   * c) 2*rows*columns in which case it is assumed the double[] provided is formed of interleaved real and imaginary values
```
Comment (gmarkall) [inline src/java/src/main/java/com/opengamma/maths/datacontainers/matrix/OGComplexDenseMatrix.java]: The next letter after A is B.

**Comment by gmarkall** [comment]
This looks good, with the exception of the letter ordering. Happy to merge once this is sorted out.

File: src/java/src/main/java/com/opengamma/maths/datacontainers/matrix/OGComplexDenseMatrix.java (hunk 1 above)
Comment (stuartarchibald) [inline src/java/src/main/java/com/opengamma/maths/datacontainers/matrix/OGComplexDenseMatrix.java]: Excellent point. Will patch.

**Comment by stuartarchibald** [comment]
Patched in adf857a

**Pull request merged**